cypress.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
set default config for `cypress` blocking recording videos on every test run. baseUrl checks the URL where the App might be running for automation tests.